src/app/public/modules/ag-grid/cell-editors/cell-editor-params.ts

```diff
@@ -0,0 +1,7 @@
+import {
+  ICellEditorParams
+} from 'ag-grid-community';
+
+export interface SkyCellEditorParams extends ICellEditorParams {
```

Comment:
What if we created a unique cell editor type for each cell component? That way, the end user would not need to "guess" what options are available to them? For example, the datepicker would have one called `SkyDatepickerCellEditorParams` that would include properties for its component?
```
export interface SkyDatepickerCellEditorParams extends ICellEditorParams {
  skyDatepickerProperties?: {
    dateFormat: string;
    startingDay: string;
    // etc.
  }
}
```

Reply:
I like that idea, them having to go look at documentation and check versions felt a little gross